datadog/data_source_datadog_monitor.go

```diff
@@ -94,8 +95,43 @@ func dataSourceDatadogMonitor() *schema.Resource {
 					},
 				},
 			},
+			"monitor_thresholds": {
+				Description: "Alert thresholds of the monitor.",
+				Type:        schema.TypeList,
+				Computed:    true,
+				MaxItems:    1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"ok": {
+							Type:     schema.TypeString,
```

Comment:
Forced to use `TypeString` to be able to differentiate between set to 0 and unset in config....